tests/ext/test_ormar.py

```diff
@@ -0,0 +1,76 @@
+import databases
+import sqlalchemy
+from fastapi import FastAPI
+from ormar import Integer, Model, ModelMeta, String
+from pytest import fixture
+
+from fastapi_pagination import Page, add_pagination
+from fastapi_pagination.ext.ormar import paginate
+from fastapi_pagination.limit_offset import Page as LimitOffsetPage
+
+from ..base import BasePaginationTestCase, SafeTestClient, UserOut
+from ..utils import faker
+
+
+@fixture(scope="session")
+def db(database_url):
+    return databases.Database(database_url)
+
+
+@fixture(scope="session")
+def metadata(database_url):
+    return sqlalchemy.MetaData()
+
+
+@fixture(scope="session")
+def User(db, metadata):
+    # weird syntax cause otherwise class definition is evaluated
+    # before fixtures are resolved, and ormar uses Metaclass that already
+    # requires sqlalchemy.Metadata() to exist
+    definition = {
+        "Meta": type("Meta", (ModelMeta,), dict(database=db, metadata=metadata)),
+        "id": Integer(primary_key=True),
+        "name": String(max_length=100),
+    }
+    User = type("User", (Model,), definition)
```

Comment:
Could we write smth like this?
```py
@fixture(scope="session")
def User(db, metadata):
    class User(Model):
        class Meta:
            tablename = "users"
            database = db
            metadata = metadata

        id = Integer(primary_key=True)
        name = String(max_length=100)
    
    return User
```

Reply:
Yeah I was wondering what the hell is going on until just recently I realized that line `metadata=metadata` was redeclaring the metadata so had to change the fixture name and it's all good. :D